src/types/markers.js

```diff
@@ -108,21 +108,23 @@ type GCMajorCompleted_Shared = {
   total_zones: number,
   total_compartments: number,
   minor_gcs: number,
-  store_buffer_overflows: number,
+  // Present when non-zero.
+  store_buffer_overflows?: number,
   slices: number,
 
   // Timing for the SCC sweep phase.
   scc_sweep_total: Milliseconds,
   scc_sweep_max_pause: Milliseconds,
 
   // The reason (if not 'None') why this GC ran non-incrementally.
-  nonincremental_reason: string,
+  nonincremental_reason?: string,
 
   // The allocated space for the whole heap before the GC started.
   allocated_bytes: number,
 
-  added_chunks: number,
-  removed_chunks: number,
+  // Only present if non-zero.
+  added_chunks?: number,
+  removed_chunks?: number,
```

Comment:
Just to make sure I haven't missed anything, this isn't used at the moment, right ? (same for store_buffer_overflows above)

Reply:
Correct. Many of these fields I added simply because they're present in the GC marker payloads already, even though I'm not making use of them in perf.html yet.